dev/doc/README.md

```diff
@@ -6,6 +6,40 @@ See the first section of [`INSTALL`](../../INSTALL).  Developers are
 recommended to use a recent OCaml version, which they can get through
 opam or Nix, in particular.
 
+## Configuring Dune caching and default verbosity
+
+There are several configuration settings that you can control globally
+with a Dune configuration file (see `man dune-config` to learn
+more). This file is generally located in `~/.config/dune/config` and
+should start with the version of the Dune language used, e.g.:
+
+```
+(lang dune 2.0)
+```
+
```

Comment:
```suggestion

`(lang dune 2.0)` is the language version setting to use for any Dune software version `2.*.*`.
```

But https://dune.readthedocs.io/en/stable/dune-files.html shows `(lang dune 2.8)`.  What's right?

Reply:
Both, dune's build language is versioned, so pick the version you want / need.

Reply:
I'll leave this one unchanged. IMHO if you want to know all the details, you can look at the Dune doc.